docs/docsite/rst/user_guide/playbooks_reuse_include_tasks_example.rst

```diff
@@ -0,0 +1,153 @@
+Include_tasks: A Worked Example
+===============================
+
+As described on the previous page, the include_tasks module enables the user to break a playbook into smaller, more manageable files that can be reused when required in plays, playbooks and roles. It is the very essence of good coding:
+
+1) Break up large programs into smaller blocks
+2) Reuse those blocks, without reinventing the wheel, as many times as required
+
+The include_tasks module is used as follows::
+
+        - name: [task name]
+          include_tasks: [list of tasks.yml]
+
+This document shows *how* the include_tasks module could be used: starting with a playbook, breaking the playbook into separate files, and then assembling a new playbook out of the new files.
+
+The Initial Playbook
+````````````````````
+Below is an example of a large, unwieldy playbook. It updates the computer, installs repos, installs some software and uninstalls some other software::
+
+        # original_playbook.yml
+        # building a RHEL desktop
+        - hosts: all
+          remote_user: ansible
+          tasks:
+          - name: update server
+            yum:
+              name: '*'
+              state: latest
+        - name: add epel-release
```

Comment:
The indentation here makes this an invalid playbook.

Reply:
Hi James, does it? Right then, I'll have to rethink the layout in that case.

Reply:
Yeah, this line `-name add epel-release` is one space too far left. It should be aligned with `-name: update server` to be read as a task.